### PR TITLE
[Mellanox] Add column drawer name and led status to output of show platform fan

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
**- What I did**

1. Add a column to display fan drawer name
2. Add a column to display fan led status

**- How I did it**

**- How to verify it**

Manual test

**- Previous command output (if the output of a command-line utility has changed)**
```
        FAN     Speed    Direction     Presence    Status          Timestamp
-----------  --------  -----------  -----------  --------  -----------------
       fan1       91%       intake      Present        OK  20200414 09:41:52
       fan2       81%       intake      Present        OK  20200414 09:41:52

```

**- New command output (if the output of a command-line utility has changed)**
```
  Drawer    LED          FAN     Speed    Direction     Presence    Status          Timestamp
--------  -----  -----------  --------  -----------  -----------  --------  -----------------
 drawer1  green         fan1       92%       intake      Present        OK  20200414 09:39:52
 drawer1  green         fan2       81%       intake      Present        OK  20200414 09:39:52

```

### CODE REVIEW

File: scripts/fanshow [hunk 1]
```diff
@@ -11,13 +11,15 @@ from swsssdk import SonicV2Connector
 from natsort import natsorted
 
 
-header = ['FAN', 'Speed', 'Direction', 'Presence', 'Status', 'Timestamp']
+header = ['Drawer', 'FAN', 'Speed', 'Direction', 'Presence', 'Status', 'LED', 'Timestamp']
```
Comment (Junchao-Mellanox) [inline scripts/fanshow]: Move 'LED' to right of 'Drawer'

Comment (liat-grozovik) [reply scripts/fanshow]: In this case you probably need to update the commit message 
Please add previous and new output for comparison

Comment (Junchao-Mellanox) [reply scripts/fanshow]: Updated.

**Review by keboliu — APPROVED** [review]

**Review by keboliu — APPROVED** [review]

**Comment by Junchao-Mellanox** [comment]
Community PR https://github.com/Azure/sonic-utilities/pull/881

**Pull request closed without merging**